opencog/asmoses/combo/main/combo-fmt-converter.cc

```diff
@@ -27,10 +27,10 @@
 #include <boost/program_options.hpp>
 #include <boost/algorithm/string.hpp>
 
-#include "../combo/iostream_combo.h"
-#include "../type_checker/type_tree.h"
-#include "../../data/table/table.h"
-#include "../../data/table/table_io.h"
+#include "opencog/asmoses/combo/combo/iostream_combo.h"
+#include "opencog/asmoses/combo/type_checker/type_tree.h"
+#include "opencog/asmoses/data/table/table.h"
+#include "opencog/asmoses/data/table/table_io.h"
```

Comment:
Well, kind of the whole point of using relative path names is that everything should continue to work fine, even if you change the name of the top-level directory (from moses to asmoses). So the idea here is that this change wasn't really needed.

I mean, I don't really care, I'm not maintaining this code, you are :-) but it seems like you made a bunch of extra work for yourself, when it could have been done more simply. (by leaving this file alone...)

Reply:
Or maybe we should not use relative includes at all and be consistent with the include statement :).

Reply:
I want to remove the relative includes but would be heaping more changes on already existing ones. So I will just revert this back.